src/main/java/ITNews/project/ITNews/service/NewsService.java

```diff
@@ -0,0 +1,99 @@
+package ITNews.project.ITNews.service;
+
+import ITNews.project.ITNews.dto.ControllerResponse;
+import ITNews.project.ITNews.dto.NewsControllerResponse;
+import ITNews.project.ITNews.dto.NewsRequest;
+import ITNews.project.ITNews.model.ImgNewsEntity;
+import ITNews.project.ITNews.model.NewsEntity;
+import ITNews.project.ITNews.model.UserEntity;
+import ITNews.project.ITNews.repository.ImgNewsRepository;
+import ITNews.project.ITNews.repository.NewsRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class NewsService {
+
+
+    private final RatingService ratingService;
+    private final NewsRepository newsRepository;
+    private final ImgNewsRepository imgNewsRepository;
+    private final CloudinaryService cloudinaryService;
+
+    public NewsControllerResponse add(NewsRequest newsRequest, UserEntity userData) {
+        return new NewsControllerResponse(newsRepository.save(createNews(newsRequest, userData)).getNewsId(),"Add News", "success", true);
```

Comment:
ohh pls check other places and remove hardcoded values

Reply:
refactoring